store/tikv/gcworker/gc_worker.go

```diff
@@ -417,11 +419,35 @@ func (w *GCWorker) checkGCInterval(now time.Time) (bool, error) {
 	return true, nil
 }
 
+// checkGCLifeTimeValid checks whether life time is small than min gc life time.
+func (w *GCWorker) checkGCLifeTimeValid(lifeTime time.Duration) (time.Duration, error) {
+	if lifeTime >= gcMinLifeTime {
```

Comment:
if the lifeTime is `10min`, which equals to `gcMinLifeTime`, and the `config.GetGlobalConfig().TiKVClient.MaxTxnTimeUse` is `20min`. This check will miss this case?

Reply:
Great, thanks. Already fix it.